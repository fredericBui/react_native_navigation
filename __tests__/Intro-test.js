import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../app/(tabs)/index';

test('renders correctly', () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});