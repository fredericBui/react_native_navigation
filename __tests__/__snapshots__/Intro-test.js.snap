// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Welcome to React navigation template !
  </Text>
</View>
`;
